README.md

```diff
@@ -40,6 +40,22 @@
 
   The Language Server for this client can be [found here](https://github.com/angelozerr/lsp4xml).
 
+## Supported VS Code settings
+
+The following settings are supported:
+
+* `xml.trace.server` : Allow user to view RPC communication through Output tab.
```

Comment:
Traces the communication between VS Code and the XML language server in the Output view.